lms/djangoapps/ccx/models.py

```diff
@@ -24,6 +26,9 @@ class CustomCourseForEdX(models.Model):
     course_id = CourseKeyField(max_length=255, db_index=True)
     display_name = models.CharField(max_length=255)
     coach = models.ForeignKey(User, db_index=True)
+    # if not empty, this field contains a compressed json serialized list of
+    # the master course modules
+    structure_json = CompressedTextField(verbose_name='Structure JSON', blank=True, null=True)
```

Comment:
the maximum number is the number of chapters of the master course. In one of our largest courses (8.MechCx) we count 22 chapters, so I do not think we can go even close to 100.